python/tk_multi_loader/delegate_publish_thumb.py

```diff
@@ -78,23 +79,13 @@ def _format_folder(self, model_index, widget):
         :param model_index: Index of the item being drawn by the delegate.
         :param widget: Qt widget created by the delegate for rendering.
         """
-        # this is a publish!
-        sg_data = shotgun_model.get_sg_data(model_index)
+
+        # Extract the Shotgun data and field value from the model index.
+        (sg_data, field_value) = model_item_data.get_item_data(model_index)
 
         header_text = ""
         details_text = ""
 
-        # this is a folder item, injected into the publish model from the entity tree
-
-        field_data = shotgun_model.get_sanitized_data(model_index,
-                                                      shotgun_model.ShotgunModel.SG_ASSOCIATED_FIELD_ROLE)
-        # examples of data:
```

Comment:
Commenting the details of `sg_data` and `field_data` was moved to `model_item_data.get_item_data(item)` in order to maintain these comments in one location and avoid the current discrepancies here and there.